crates/torii/core/src/processors/store_set_record.rs

```diff
@@ -53,8 +53,10 @@ where
         let keys_end: usize = keys_start + usize::from(u8::try_from(event.data[NUM_KEYS_INDEX])?);
         let keys = event.data[keys_start..keys_end].to_vec();
 
-        let values_start = keys_end + 2;
-        let values_end: usize = values_start + usize::from(u8::try_from(event.data[keys_end + 1])?);
+        // keys_end is already the length of the values array.
+
+        let values_start = keys_end + 1;
+        let values_end: usize = values_start + usize::from(u8::try_from(event.data[keys_end])?);
```

Comment:
@tarrencev I would appreciate you having a look to that if you have a chance. It seemed weird to me to have 1 offset difference.

The event has not changed, so I'm trying to figure out which side effect could have change that. Because before hand tests were passing.

Other thing I'm wondering. As the `world.uuid()` is 0 at first, some computation on indexes may have gone wrong without hurting the tests as lot's of 0 are in the event.